manager/controlapi/common.go

```diff
@@ -84,3 +87,17 @@ func validateDriver(driver *api.Driver) error {
 
 	return nil
 }
+
+// combineErrors combines multi errors into a single one.
+func combineErrors(errs []error) error {
+	if len(errs) == 0 {
+		return nil
+	}
+
+	var errStr []string
+	for _, err := range errs {
+		errStr = append(errStr, err.Error())
+	}
+
+	return errors.New(strings.Join(errStr, " "))
```

Comment:
Maybe this should use a separator character like `;` or `,`. If the errors are joined with spaces, it will be hard for the user to parse a set of several errors.